packages/web-app-files/tests/unit/components/SideBar/Shares/FileLinks.spec.js

```diff
@@ -4,6 +4,7 @@ import GetTextPlugin from 'vue-gettext'
 import DesignSystem from 'owncloud-design-system'
 import { mount, shallowMount, createLocalVue } from '@vue/test-utils'
 import FileLinks from '@files/src/components/SideBar/Shares/FileLinks.vue'
+import CreateForm from "../../../../../src/components/SideBar/Shares/Links/CreateForm";
```

Comment:
Please use the `@files` import shorthand (see line above). Also, linter complains about the trailing `;` ;-)

Reply:
:+1: